test-d/types/utils.test-d.ts

```diff
@@ -178,10 +178,10 @@ type SomeItemData = Item.Data<{}>;
 class SomeItem extends Item<SomeItemData> {}
 const someItem = new SomeItem();
 const someItemData = duplicate<SomeItem, 'lenient'>(someItem);
-SomeItem.create<SomeItem>(someItemData);
+SomeItem.create<SomeItem, typeof someItemData>(someItemData); // TODO: Investigate why `U` isn't properly getting inferred
 
 type SomeActorData = Actor.Data<{}, SomeItemData>;
 class SomeActor extends Actor<SomeActorData, SomeItem> {}
 const someActor = new SomeActor();
 const someActorData = duplicate<SomeActor, 'lenient'>(someActor);
-SomeActor.create<SomeActor>(someActorData);
+SomeActor.create<SomeActor, typeof someActorData>(someActorData);
```

Comment:
You can use `SomeActorData` here directly instead of using `typeof`.

Reply:
Hopefully I can make it clean again after rebasing